x-pack/plugins/session_view/public/components/process_tree_node/index.tsx

```diff
@@ -179,20 +200,31 @@ export function ProcessTreeNode({
             >
               <FormattedMessage
                 id="xpack.sessionView.execUserChange"
-                defaultMessage="Root escalation"
+                defaultMessage="Exec user change: Root"
```

Comment:
I think we might want to change this behavior. Currently it'll only show if a user switches to root. but there can be other users on a system with escalated priveleges, so probably best to show any user change. e.g  process.user.id != process.parent.user.id